app/models/person.rb

```diff
@@ -9,8 +9,11 @@ class Person < ApplicationRecord
   has_many :offers, inverse_of: :person
   has_many :listings
 
-  has_many :matches, as: :receiver
-  has_many :matches, as: :provider
+  has_many :communication_logs
+  has_many :listings
```

Comment:
Maybe we want asks and offers separated here later instead of `has_many :listings`?